<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>переход</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>\</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>10</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>1</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>тип</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>дата</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>создать</string>
     </property>
     <addaction name="action_8"/>
     <addaction name="action_9"/>
    </widget>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="menu_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>настройки</string>
    </property>
    <addaction name="action_temp"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>сортировка</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>переименовать</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>папку</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>файл</string>
   </property>
  </action>
  <action name="action_temp">
   <property name="text">
    <string>temp</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>чистить папку при каждом новом поиске</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
